libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -401,11 +401,17 @@ namespace SohImGui {
             EnhancementRadioButton("German", "gLanguages", 1);
             EnhancementRadioButton("French", "gLanguages", 2);
         */
-        int val = CVar_GetS32(cvarName.c_str(), 0);
-        if (ImGui::RadioButton(text.c_str(), id == val)) {
-            CVar_SetS32(cvarName.c_str(), (int)id);
+        std::string make_invisible = "##";
+        make_invisible += text;
+        make_invisible += cvarName;
```

Comment:
i think this can be a one-liner
```
std::string make_invisible = "##" + text + cvarName;
```

Reply:
Nope, you can't add two `const char*`

Reply:
Is there a reason they were changed from strings in this PR? L393

Reply:
yes, #359 
this follow this idea.
cleanup is important :)

Reply:
Cool, then the 3 lines make sense!